rules/linux/macos_clear_system_logs.yml

```diff
@@ -0,0 +1,25 @@
+title: Indicator Removal on Host - Clear Mac System Logs
+id: acf61bd8-d814-4272-81f0-a7a269aa69aa
+status: experimental
+description: Detects deletion of local audit logs
+author: remotephone, oscd.community
+date: 2020/10/11
+references:
+    - https://github.com/redcanaryco/atomic-red-team/blob/master/atomics/T1070.002/T1070.002.md
+logsource:
+    product: macos
+    category: process_creation
+detection:
+    selection:
+      - ProcessName: 'rm' 
+        CommandLine|contains:
+            - '-rf /var/log'
+            - '-rf /private/var/log'
+            - '-rf /Users/*/Library/Logs/'
```

Comment:
there are a few problems with that section. let me explain them one by one:

1. `-rf` is not the only option to recursively remove some files and folders. for example, you can use `-fr` with the same result. and also you can add other optional flags, and bypass the rule this way. so better not rely on these flags in this scenario

2. spaces. they also make the rule easily bypassed. at the same time, osquery and santa split arguments into separate elements of the list (as you can see on a log sample above). so you will not see any spaces there. we better get rid of them

3. asterisk. that's also a problem for some backends. better to avoid using them. 

here is my proposal:

```
    selection1:
      - ProcessName|endswith: '/rm'
    selection2:
        CommandLine|contains:
            - '/var/log' # no need in `/private/var/log` as it is already covered by this one 
    selection3:
        CommandLine|contains|all:
            - '/Users/'
            - '/Library/Logs/'
    condition: selection1 and (selection2 or selection3)
```

Reply:
Thank you, that does look good and makes sense how that's less likely to be evaded. I have brought in the recommended changes in the latest commit.